### ISSUE TITLE
Suffixing fraction separator is removed on editing text before it.

### ISSUE THREAD

**Issue by oscbyspro** [oscbyspro/DiffableTextViews#94]
It takes two keystrokes to remove a suffixing fraction separator. It's a bit odd, honestly, but it should be easy to fix.

**Comment by oscbyspro** [comment]
I found the problem. It's kind funny. I made a functional helper method in Bool.swift, to reduce the boilerplate when deciding to merge new values in Context.swift. This is the culprit.

 ```swift
@inlinable @discardableResult func on(_ value: Bool, perform action: () throws -> Void) rethrows -> Bool {
    if value { try action() }; return self
}
```

As you can see, I forgot to check `self == value`. This method is largely useless, and doesn't really improve readability at all, but I thought it was cool and I felt cute, so I added it. It will be removed on the next patch, later today.

**Comment by oscbyspro** [comment]
Solved in v4.0.1, 76799df84810d2551ccac0dc7d8e55bc92c5a263
